main.py

```diff
@@ -1,7 +1,53 @@
 import init_django_orm  # noqa: F401
 
 from django.db.models import QuerySet
+from db.models import Actor, Genre
 
 
 def main() -> QuerySet:
-    pass
+    genre_to_create = ["Western", "Action", "Dramma"]
```

Comment:
The genre 'Dramma' seems to be a typo. It is later updated to 'Drama', which suggests the initial creation should also use 'Drama' to avoid unnecessary updates.